prov/rxm/src/rxm_ep.c

```diff
@@ -409,16 +409,16 @@ static int rxm_ep_rx_queue_init(struct rxm_ep *rxm_ep)
 				  rxm_ep->rxm_info->rx_attr->size,
 				  RXM_RECV_QUEUE_MSG);
 	if (ret)
-		return ret;
+		goto err_recv;
```

Comment:
This change is wrong.  We failed to init recv_queue, so we can't call close on it.